tests/test_optimizers.py

```diff
@@ -2,8 +2,9 @@
 
 from diffpy.snmf.optimizers import get_weights
 
-tm = [
-    ([[[1, 0], [0, 1]], [1, 1], [0, 0], [1, 1]], [0, 0]),
+test_matrix = [
+    # ([stretched_component_gram_matrix, linear_coefficient, lower_bound, upper_bound], expected)
+    ([[[1, 0], [0, 1]], [1, 1], 0, 0], [0, 0]),
```

Comment:
Fixing syntax bug - here `0, 0]` refers to `lower_bound` and `upper_bound`, respectively.